### PR TITLE
DOC add docs on broken packages

### PR DESCRIPTION
**Pull request by beckermr**
I have updated the documentation on broken packages to match our latest policies.

cc @conda-forge/core for review!

### CODE REVIEW

**Comment by beckermr** [comment]
merging. happy to fix any issues ppl find!

File: src/maintainer/updating_pkgs.rst [hunk 1]
```diff
@@ -201,11 +201,16 @@ Removing broken packages
...
+the `repo data patches feedstock <https://github.com/conda-forge/conda-forge-repodata-patches-feedstock>`__.
+To change the repo data for your package, make a PR on the feedstock.
+
+If instead the actual contents of the package are broken, the following steps will
+remove broken packages from the ``main`` channel:
 
 1. Locate the paths to broken files on `anaconda.org <https://anaconda.org>`__, by searching for the conda-forge package and switching to the files tab.
-2. Fork `conda-forge/cf-mark-broken <https://github.com/conda-forge/cf-mark-broken>`__ and add a new text file in the ``pkgs`` directory.
-3. Add the broken files to the new text file, one path per line. See `pkgs/example.txt <https://github.com/conda-forge/cf-mark-broken/blob/master/pkgs/example.txt>`__ for an example file.
+2. Fork `conda-forge/admin-requests <https://github.com/conda-forge/admin-requests>`__ and add a new text file in the ``broken`` directory.
```
Comment (jakirkham) [inline src/maintainer/updating_pkgs.rst:212]: Oh. So we don’t use `cf-mark-broken` anymore? Why is that?

Comment (beckermr) [reply src/maintainer/updating_pkgs.rst:212]: It has been renamed to admin-requests.

Comment (jakirkham) [reply src/maintainer/updating_pkgs.rst:212]: Gotcha. Are there other workflows we are planning to include there?

Comment (beckermr) [reply src/maintainer/updating_pkgs.rst:212]: Yup. Removing broken labels is already done. Resetting feedstock tokens is the next one to do after that.

Comment (jakirkham) [reply src/maintainer/updating_pkgs.rst:212]: Ah ok. Thanks for clarifying Matt 🙂

**Pull request merged**